Custom/Mails/mailhistoryunit.cpp

```diff
@@ -0,0 +1,82 @@
+#include "mailhistoryunit.h"
+#include "Custom/Mails/ui_mailhistoryunit.h"
+#include "mailhistoryunit.h"
```

Comment:
"mailhistoryunit.h" including current file twice

Reply:
Already noticed by [chrkspln](https://github.com/chrkspln) and already fixed! Thanks for noticing!